### ISSUE TITLE
Run "bazel build --config=mkl --config=opt //tensorflow/tools/pip_package:build_pip_package --verbose_failures" failed

### ISSUE THREAD

**Issue by zhucan** [tensorflow/tensorflow#55524]
**System information**
- CentOS Linux release 7.8.2003 (Core)

**Describe the current behavior**
bazel build --config=mkl --config=opt //tensorflow/tools/pip_package:build_pip_package --verbose_failures

logs:
![image](https://user-images.githubusercontent.com/13997135/162117884-3aa73da3-d0bc-4a37-b991-c7d49f91697e.png)

**Comment by gadagashwini** [comment]
@zhucan,
Hi, Thanks for reporting this issue. 
Could you provide the configuration  details like Bazel, Tensorflow version. 

Run `bazel clean --expunge`, forcing bazel to re-download and build each dependency to avoid issue. Also sync the bazel build `bazel sync`. Thanks!

**Comment by zhucan** [comment]
@gadagashwini Thanks for your response. it's ok for me. But there is no "build doc" for centos7.8. And there is no docs build tensorflow(v2.8.0) with verbs?

**Comment by zhucan** [comment]
For tensorflow(v2.8.0) on the centos7.8, like this:

1. sudo yum install python3-devel python3-pip
2. pip install -U --user pip numpy wheel
3. pip install -U --user keras_preprocessing --no-deps
4. ./configure
5. bazel build --distdir=/home/bazel_package //tensorflow/tools/pip_package:build_pip_package
it's ok for me, but I don't know how to build the tensorflow with verb. There is no config for it.
![image](https://user-images.githubusercontent.com/13997135/162182353-b2ce9f8c-821d-4c40-a5e9-dedcff3aabd6.png)

![image](https://user-images.githubusercontent.com/13997135/162182527-af81bfbf-1ec1-4c85-8e73-e92f9d61dc7d.png)

**Comment by zhucan** [comment]
@gadagashwini Can you help me to look at this issue.https://github.com/tensorflow/networking/issues/40

**Comment by zhucan** [comment]
ERROR: /home/deeproute/workspace/tensorflow/tensorflow/compiler/mlir/tensorflow/BUILD:432:15: Compiling tensorflow/compiler/mlir/tensorflow/ir/tf_ops_a_m.cc failed: (Exit 4): gcc failed: error executing command 
  (cd /root/.cache/bazel/_bazel_root/b0f1c7b7aacab844a054d97356b0d4dd/execroot/org_tensorflow && \
  exec env - \
    PATH=/root/.cargo/bin:/sbin:/bin:/usr/sbin:/usr/bin \
    PWD=/proc/self/cwd \
    PYTHON_BIN_PATH=/bin/python3 \
    PYTHON_LIB_PATH=/usr/lib/python3.6/site-packages \
    TF2_BEHAVIOR=1 \
  /bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' -DNDEBUG -ffunction-sections -fdata-sections '-std=c++11' -MD -MF bazel-out/k8-opt/bin/tensorflow/compiler/mlir/tensorflow/_objs/tensorflow_ops_a_m/tf_ops_a_m.pic.d '-frandom-seed=bazel-out/k8-opt/bin/tensorflow/compiler/mlir/tensorflow/_objs/tensorflow_ops_a_m/tf_ops_a_m.pic.o' -fPIC '-DLLVM_ON_UNIX=1' '-DHAVE_BACKTRACE=1' '-DBACKTRACE_HEADER=<execinfo.h>' '-DLTDL_SHLIB_EXT=".so"' '-DLLVM_PLUGIN_EXT=".so"' '-DLLVM_ENABLE_THREADS=1' '-DHAVE_DEREGISTER_FRAME=1' '-DHAVE_LIBPTHREAD=1' '-DHAVE_PTHREAD_GETNAME_NP=1' '-DHAVE_PTHREAD_GETSPECIFIC=1' '-DHAVE_PTHREAD_H=1' '-DHAVE_PTHREAD_SETNAME_NP=1' '-DHAVE_REGISTER_FRAME=1' '-DHAVE_SETENV_R=1' '-DHAVE_STRERROR_R=1' '-DHAVE_SYSEXITS_H=1' '-DHAVE_UNISTD_H=1' -D_GNU_SOURCE '-DHAVE_LINK_H=1' '-DHAVE_LSEEK64=1' '-DHAVE_MALLINFO=1' '-DHAVE_SBRK=1' '-DHAVE_STRUCT_STAT_ST_MTIM_TV_NSEC=1' '-DLLVM_NATIVE_ARCH="X86"' '-DLLVM_NATIVE_ASMPARSER=LLVMInitializeX86AsmParser' '-DLLVM_NATIVE_ASMPRINTER=LLVMInitializeX86AsmPrinter' '-DLLVM_NATIVE_DISASSEMBLER=LLVMInitializeX86Disassembler' '-DLLVM_NATIVE_TARGET=LLVMInitializeX86Target' '-DLLVM_NATIVE_TARGETINFO=LLVMInitializeX86TargetInfo' '-DLLVM_NATIVE_TARGETMC=LLVMInitializeX86TargetMC' '-DLLVM_NATIVE_TARGETMCA=LLVMInitializeX86TargetMCA' '-DLLVM_HOST_TRIPLE="x86_64-unknown-linux-gnu"' '-DLLVM_DEFAULT_TARGET_TRIPLE="x86_64-unknown-linux-gnu"' -D__STDC_LIMIT_MACROS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -DEIGEN_MPL2_ONLY '-DEIGEN_MAX_ALIGN_BYTES=64' -iquote. -iquotebazel-out/k8-opt/bin -iquoteexternal/llvm-project -iquotebazel-out/k8-opt/bin/external/llvm-project -iquoteexternal/llvm_terminfo -iquotebazel-out/k8-opt/bin/external/llvm_terminfo -iquoteexternal/llvm_zlib -iquotebazel-out/k8-opt/bin/external/llvm_zlib -iquoteexternal/com_google_absl -iquotebazel-out/k8-opt/bin/external/com_google_absl -iquoteexternal/nsync -iquotebazel-out/k8-opt/bin/external/nsync -iquoteexternal/eigen_archive -iquotebazel-out/k8-opt/bin/external/eigen_archive -iquoteexternal/gif -iquotebazel-out/k8-opt/bin/external/gif -iquoteexternal/libjpeg_turbo -iquotebazel-out/k8-opt/bin/external/libjpeg_turbo -iquoteexternal/com_google_protobuf -iquotebazel-out/k8-opt/bin/external/com_google_protobuf -iquoteexternal/com_googlesource_code_re2 -iquotebazel-out/k8-opt/bin/external/com_googlesource_code_re2 -iquoteexternal/farmhash_archive -iquotebazel-out/k8-opt/bin/external/farmhash_archive -iquoteexternal/fft2d -iquotebazel-out/k8-opt/bin/external/fft2d -iquoteexternal/highwayhash -iquotebazel-out/k8-opt/bin/external/highwayhash -iquoteexternal/zlib -iquotebazel-out/k8-opt/bin/external/zlib -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinAttributeInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinAttributesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinDialectIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinLocationAttributesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinOpsIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinTypeInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/BuiltinTypesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/CallOpInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/CastOpInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/InferTypeOpInterfaceIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/OpAsmInterfaceIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/RegionKindInterfaceIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/SideEffectInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/SubElementInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/SymbolInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/TensorEncodingIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/ParserTokenKinds -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/ControlFlowInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/DerivedAttributeOpInterfaceIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/LoopLikeInterfaceIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/ArithmeticBaseIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/ArithmeticCanonicalizationIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/ArithmeticOpsIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/VectorInterfacesIncGen -Ibazel-out/k8-opt/bin/external/llvm-project/mlir/_virtual_includes/StandardOpsIncGen -isystem external/llvm-project/mlir/include -isystem bazel-out/k8-opt/bin/external/llvm-project/mlir/include -isystem external/llvm-project/llvm/include -isystem bazel-out/k8-opt/bin/external/llvm-project/llvm/include -isystem external/nsync/public -isystem bazel-out/k8-opt/bin/external/nsync/public -isystem third_party/eigen3/mkl_include -isystem bazel-out/k8-opt/bin/third_party/eigen3/mkl_include -isystem external/eigen_archive -isystem bazel-out/k8-opt/bin/external/eigen_archive -isystem external/gif -isystem bazel-out/k8-opt/bin/external/gif -isystem external/com_google_protobuf/src -isystem bazel-out/k8-opt/bin/external/com_google_protobuf/src -isystem external/farmhash_archive/src -isystem bazel-out/k8-opt/bin/external/farmhash_archive/src -isystem external/zlib -isystem bazel-out/k8-opt/bin/external/zlib -w -DAUTOLOAD_DYNAMIC_KERNELS '-std=c++14' -fno-canonical-system-headers -Wno-builtin-macro-redefined '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__="redacted"' -c tensorflow/compiler/mlir/tensorflow/ir/tf_ops_a_m.cc -o bazel-out/k8-opt/bin/tensorflow/compiler/mlir/tensorflow/_objs/tensorflow_ops_a_m/tf_ops_a_m.pic.o)
Execution platform: @local_execution_config_platform//:platform
gcc: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://bugzilla.redhat.com/bugzilla> for instructions.
Target //tensorflow/tools/pip_package:build_pip_package failed to build
INFO: Elapsed time: 1871.483s, Critical Path: 251.24s
INFO: 1847 processes: 299 internal, 1548 local.
FAILED: Build did NOT complete successfully
@gadagashwini

**Comment by gadagashwini** [comment]
@zhucan, For Cent OS, Tensorflow build instructions mentioned for [Ubuntu](https://www.tensorflow.org/install/source#ubuntu) can be used. 
You can build Tensorflow with VERBS config
`--config=verbs # Build with libverbs support.`. Thanks!

**Comment by zhucan** [comment]
The "--config=verbs" is not suitable for tensorflow 2.8.0 . @gadagashwini

**Comment by gadagashwini** [comment]
@zhucan, 

Tensorflow 2.0 build supports `--config=verbs`.
`config_info_line('verbs', 'Build with libverbs support.')` 
For more information take a look [here](https://github.com/tensorflow/tensorflow/blob/r2.0/configure.py#L1581). Thanks!

**Comment by zhucan** [comment]
@gadagashwini I mean tensorflow2.8 not 2.0

**Comment by zhucan** [comment]
https://github.com/tensorflow/tensorflow/blob/r2.8/configure.py#L1581 @gadagashwini no verbs config.

**Comment by gadagashwini** [comment]
@zhucan, Tensorflow v2.8  has no `--config==verbs`.
Tensorflow v2.0 supports `--config==verbs`

**Comment by zhucan** [comment]
I know, so I want to know how to compile tensorflow2.8.0 with verbs..... @gadagashwini

**Comment by mihaimaruseac** [comment]
What is the commit you are trying to build?

Can you use proper markdown formatting on the errors so that they are easier to read?

Can you explain what you want to do? It seems this is a X vs Y type of problem
